src/components/Camera/index.js

```diff
@@ -38,36 +38,29 @@ type CameraErrorType = {
 }
 
 class CameraError extends React.Component<CameraErrorType> {
-  fileInput = null
 
   componentDidMount () {
     if (this.props.cameraError.type === 'error') {
       this.props.trackScreen('camera_error')
     }
   }
 
-  handleFallback = (event) => {
-    if (this.fileInput) { this.props.onUploadFallback(this.fileInput.files[0]) }
-    // Remove target value to allow upload of the same file if needed
-    event.target.value = null
-  }
-
-  onFallbackClick = () => {
-    if (this.fileInput) { this.fileInput.click(); }
+  handleFileInputClick = () => {
     if (this.props.cameraError.type === 'warning') {
       this.props.trackScreen('fallback_triggered')
     }
   }
 
   errorInstructions = (text) =>
-    <span onClick={this.onFallbackClick} className={style.errorLink}>
+    <CustomFileInput
+      className={style.errorLink}
+      onFileSelected={this.props.onUploadFallback}
+      onClick={this.handleFileInputClick}
+      accept="image/*"
+      capture={ this.props.method === 'face' ? 'user' : true }
```

Comment:
FYI I think that the capture facing mode only works on iOS. On Android, it seems like it  returns the last used camera mode

Reply:
Yeah, I think we discuss this briefly when working on #414. I didn't change this BTW